synfig-studio/src/gui/docks/dockbook.cpp

```diff
@@ -168,6 +168,7 @@ DockBook::refresh_tab(Dockable* dockable)
 	);
 
 	set_tab_label(*dockable, *label);
+	label->set_halign(Gtk::ALIGN_CENTER);
```

Comment:
This method affects all tabs, right? Not only canvas views.

Reply:
Yes, that's right

Reply:
With the last commit only document tabs are affected.